architecture/routing.adoc

```diff
@@ -10,21 +10,18 @@
 toc::[]
 
 == Overview
-Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information
-directly to the router (HTTP, HTTPS, and TLS with SNI). Routers subscribe to configuration changes and automatically
-update themselves with new configuration, and routers may be containerized or virtual (converting those changes to API calls to a system like an F5).
+Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information directly to the router (HTTP, HTTPS, and TLS with SNI). Routers subscribe to a configuration and automatically update themselves with any changes, and may be containerized or virtual (converting those changes to API calls to a system like an F5).
```

Comment:
Implementation detail that may be a little confusing, not sure if we want to make the distinction or not....In the current implementation of the router it doesn't actually forward traffic to the service.  By that, I mean, it does not use the service ip/port when forwarding traffic.  It forwards traffic to the endpoints that back the service.  It only uses the service selector to watch the endpoints that back the service.  (It's ultimately the same thing, but bypasses the service load balancing logic and replaces it with its own. Maybe it isn't worth mentioning)